Voeg FAQ toe als FAQ pagina (#771)
nl-design-system/documentatie: docs/project/faq.mdx

---
title: Veelgestelde vragen
hide_title: true
hide_table_of_contents: true
sidebar_label: Veelgestelde vragen
pagination_label: Veelgestelde vragen
description: Een overzicht van vragen die ons regelmatig gesteld worden.
keywords:
  - faq
  - nl design system
---

# Veelgestelde vragen

Op deze pagina geven we antwoord op vragen die regelmatig aan het kernteam van NL Design System gesteld worden.

- [Wat is het NL Design System?](#wat-is-het-nl-design-system)
- [Hoe werkt het NL Design System en waarom?](#hoe-werkt-het-nl-design-system-en-waarom)
- [Wat is het estafettemodel? Hoe werkt dat model?](#wat-is-het-estafettemodel-hoe-werkt-dat-model)
- [Waarom maken jullie niet gewoon zelf componenten, richtlijnen en ontwerpen en stellen die beschikbaar?](#waarom-maken-jullie-niet-gewoon-zelf-componenten-richtlijnen-en-ontwerpen-en-stellen-die-beschikbaar)
- [Waarom hebben jullie een community? Wat doet die voor mij?](#waarom-hebben-jullie-een-community-wat-doet-die-voor-mij)

> - [Welke NLDS-richtlijnen en componenten kan ik nu direct gebruiken om mijn website digitaal, gebruiksvriendelijk en toegankelijk te krijgen?](#welke-nlds-richtlijnen-en-componenten-kan-ik-nu-direct-gebruiken-om-mijn-website-digitaal-gebruiksvriendelijk-en-toegankelijk-te-krijgen)

Author: ```suggestion
- [Welke richtlijnen en componenten kan ik nu direct gebruiken om mijn website digitaal, gebruiksvriendelijk en toegankelijk te krijgen?](#welke-nlds-richtlijnen-en-componenten-kan-ik-nu-direct-gebruiken-om-mijn-website-digitaal-gebruiksvriendelijk-en-toegankelijk-te-krijgen)
```

## Waarom maken jullie niet gewoon zelf componenten en patronen?

We ontwikkelen de onderdelen niet zelf, omdat we de doelen van deelnemende organisaties centraal willen stellen. En, nog belangrijker, de behoeften van hun gebruikers. Doordat specialisten in de community samenwerken stimuleren we vanaf het begin breed draagvlak. Zo worden oplossingen direct in de praktijk gebruikt en kunnen ze gelijk met de juiste doelgroep worden getest.

## Waarom hebben jullie een community? Wat doet die voor mij?

Overheidsorganisaties hebben veel overlap in de online diensten die ze aanbieden. Die overlap maakt samenwerken logisch en dat doen wij als community. Want als we dezelfde uitdagingen hebben, kunnen we vast ook van elkaar leren wat betreft oplossingen.

Als organisatie die meewerkt aan NL Design System beslis je dus mee over nieuwe ‘bouwblokken’, profiteer je van wat er al is en maak je gebruik van de expertise van het kernteam én de rest van de community.

## Welke NLDS-richtlijnen en componenten kan ik nu direct gebruiken om mijn website digitaal, gebruiksvriendelijk en toegankelijk te krijgen?

> In de community zijn al verschillende richtlijnen en componenten te vinden. Voor vragen, meld je gerust in de [Slack-community van Code for NL](https://praatmee.codefor.nl/) (kanaal `#nldesignsystem`).

Author: ```suggestion
In de community zijn al veel verschillende [richtlijnen](/richtlijnen) en [componenten](/componenten) te vinden. De richtlijnen zijn direct op deze website te vinden, voor de componenten kun je terecht bij de organisaties waar ze gemaakt zijn. Op de pagina [componenten](/componenten) vind je links naar de relevante Storybooks en Figma-bestanden.

Voor vragen, design of development: meld je gerust in de [Slack-community van Code for NL](https://praatmee.codefor.nl/) (kanaal `#nldesignsystem`).
```

Meedoen is gratis en kan stap voor stap. De kosten voor organisaties zitten in het beschikbaar stellen van resources en de tijd vrijmaken waarin ze samenwerken.

## Zijn er al andere organisaties die meedoen?

NL Design System wordt gebruikt in verschillende projecten van overheidsorganisaties door heel Nederland. Veel van deze projecten zijn in ontwikkeling, sommige projecten staan live. Op de pagina [‘Wie doet mee?’](/community/wie-doet-mee/) vind je een overzicht van organisaties die zelf en/of via hun leveranciers meedoen met NL Design System.

Meedoen kan op allerlei manieren, van alleen gebruiken tot zelf meebouwen:

- **Gebruik de bouwstenen**: gebruik [richtlijnen](/richtlijnen/introductie), [componenten](/componenten) of [onderzoek](/onderzoek) voor je websites en applicaties.
- **Doe kennis op**: kom naar bijeenkomsten als de tweewekelijkse Heartbeat, Design Systems Week of hackatons. Of word lid van onze [Slack community bij CodeForNL](https://praatmee.codefor.nl/) (kanaal: #nl-design-system).
- **Werk mee**: deel je eigen richtlijnen, componenten of onderzoek, of geef feedback op de bestaande bouwstenen.

## Hoe is security geregeld binnen NLDS?

> Het kernteam beheert de [repository's](https://github.com/orgs/nl-design-system/repositories) waarin de diverse NL Design System projecten publiek worden ontsloten, zoals thema's en organisatie-specifieke design systems.

Reviewer: ```suggestion
Het kernteam helpt de community om hun [repository's](https://github.com/orgs/nl-design-system/repositories) vers en veilig te houden, zodat iedereen gerust gebruik kan maken van de componenten en thema's.

- Alle code wordt verplicht door een tweede teamlid gecontroleerd.
- Elke dag worden backups gemaakt van de NL Design System projecten op GitHub.
- Elke maand werken we samen om updates te installeren.
- De updates gaan niet automatisch, alle wijzigingen worden door twee personen gecontroleerd.
- Organisaties die werken op [github.com/nl-design-system](github.com/nl-design-system) worden [geholpen met infrastructuur](http://github.com/nl-design-system/terraform) die toegang tot de code en de integriteit van de code bewaakt.
- Twee-factor-authenticatie (2FA) is verplicht voor alle bijdragen aan NL Design System.
- Alle code controleren we automatisch op secure coding practices, en voldoen aan die regels is verplicht.
```

- [Welke NLDS-richtlijnen en componenten kan ik nu direct gebruiken om mijn website digitaal, gebruiksvriendelijk en toegankelijk te krijgen?](#welke-nlds-richtlijnen-en-componenten-kan-ik-nu-direct-gebruiken-om-mijn-website-digitaal-gebruiksvriendelijk-en-toegankelijk-te-krijgen)
- [Ik wil NL Design System gaan gebruiken voor mijn toegankelijkheidsopgave maar waar moet ik beginnen?](#ik-wil-nl-design-system-gaan-gebruiken-voor-mijn-toegankelijkheidsopgave-maar-waar-moet-ik-beginnen)
- [Mijn leverancier heeft vragen over NL Design System. Naar wie van jullie kan ik hem verwijzen?](#mijn-leverancier-heeft-vragen-over-nl-design-system-naar-wie-van-jullie-kan-ik-hem-verwijzen)
- [Ik wil mijn manager voorstellen om mee te doen met NL Design System. Wat zijn de kosten en doorlooptijd? Waar moet ik zelf voor zorgen?](#ik-wil-mijn-manager-voorstellen-om-mee-te-doen-met-nl-design-system-wat-zijn-de-kosten-en-doorlooptijd-waar-moet-ik-zelf-voor-zorgen)
- [Zijn er al andere organisaties die meedoen?](#zijn-er-al-andere-organisaties-die-meedoen)
- [Hoe is security geregeld binnen NLDS?](#hoe-is-security-geregeld-binnen-nlds)

## Wat is het NL Design System?

> NL Design System is een project dat specialisten van verschillende overheidsorganisaties laat samenwerken aan concrete oplossingen voor websites en applicaties. Denk bijvoorbeeld aan code, ontwerp en richtlijnen.

Reviewer 2: Hoewel ik deze suggestie goed vind, vind ik de zin te lang. Kunnen we hier losse zinnen van maken @hidde ? Bijvoorbeeld: 
NL Design System is een project dat meer samenwerking mogelijk maakt van specialisten die werken aan concrete oplossingen voor websites en applicaties, bij verschillende overheidsorganisaties. Ze delen en ontwikkelen herbruikbare code, ontwerpen en richtlijnen.

> Het doel: organisaties hoeven minder vaak hetzelfde wiel uit te vinden. Zo worden online overheidsomgevingen consistenter, gebruiksvriendelijker en toegankelijker.

Author: ```suggestion
Het doel: organisaties hoeven minder vaak hetzelfde wiel uit te vinden, en hebben meer tijd om gebruikers centraal te stellen. Zo worden online overheidsomgevingen consistenter, gebruiksvriendelijker en toegankelijker.
```

> ## Hoe werkt het NL Design System en waarom?

Author: ```suggestion
## Hoe werkt het NL Design System?
```

Het NL Design System bestaat uit verschillende soorten onderdelen:

- [Richtlijnen](/richtlijnen/introductie): adviezen over verschillende onderwerpen, zoals het [maken van formulieren](/richtlijnen/formulieren/).

> - [Componenten](/componenten): bouwblokken waarmee websites en applicaties kunnen worden opgebouwd, zoals knoppen, invulvelden en tabellen.

Author: ```suggestion
- [Componenten](/componenten): bouwblokken waarmee websites en applicaties worden opgebouwd, zoals knoppen, invulvelden en tabellen.
```

> - [Patronen](/voorbeelden/patronen): voorbeelden van hoe componenten samen kunnen worden gebruikt voor specifieke taken die gebruikers willen doen, op basis van inzichten uit [gebruikersonderzoek](/voorbeelden/onderzoek).

Author: ik twijfel (:D) of we het weg moeten halen… wel belangrijk detail dat het alleen mogelijke richtingen en niet per se _de_ richting.

> Er is ook een [handboek](/handboek/introductie), documentatie over het gebruik van NL Design System.

Author: nice!

Door zo'n uitgebreid stappenplan te doorlopen, zorgen we voor een aantal zekerheden:

- **Goed onderzocht**: we nemen gebruikersonderzoek als basis voor bruikbare richtlijnen, componenten, patronen en betere toegankelijkheid.
- **Publiek beschikbaar**: we maken open source werken makkelijker, waardoor samenwerken makkelijk wordt.
- **Samen ontwikkeld**: oplossingen worden gebouwd en gebruikt door een actieve community.
- **Breed gedragen**: onderdelen zijn bij meerdere organisaties gebruikt en daardoor getest met zoveel mogelijk verschillende use cases.

## Waarom maken jullie niet gewoon zelf componenten en patronen?

We ontwikkelen de onderdelen niet zelf, omdat we de doelen van deelnemende organisaties centraal willen stellen. En, nog belangrijker, de behoeften van hun gebruikers. Doordat specialisten in de community samenwerken stimuleren we vanaf het begin breed draagvlak. Zo worden oplossingen direct in de praktijk gebruikt en kunnen ze gelijk met de juiste doelgroep worden getest.

## Waarom hebben jullie een community? Wat doet die voor mij?

> Overheidsorganisaties hebben veel overlap in de online diensten die ze aanbieden. Die overlap maakt samenwerken logisch en dat doen wij als community. Want als we dezelfde uitdagingen hebben, kunnen we vast ook van elkaar leren wat betreft oplossingen.

Reviewer: De community begon in 2017 toen werd opgemerkt dat er veel overlap is in de online diensten die overheidsorganisaties aanbieden, dat veel ontwikkelwerk en gebruikersonderzoek dubbel wordt gedaan, en toch slechts een fractie van de overheidssites toegankelijk en gebruiksvriendelijk is. Zou veel meer samenwerken en de oplossingen verzamelen niet een goede oplossing zijn?

De groeiende community is nog altijd het hart van dit project. Sinds 2020 heeft het Ministerie van Binnenlandse Zaken budget beschikbaar gesteld voor een kernteam dat de community ondersteunt, zodat meer organisaties makkelijker kunnen meewerken.

Reviewer: I guess wat ik hier bedoel is dat het project niet de community heeft gemaakt, maar andersom :)

Author: het is goede context, maar lijkt mij teveel detail voor deze pagina, werkt denk ik beter in een presentatie?

Reviewer 2: Toch vind ik die eerste zin die @Robbert noemt wel een goede (en sterke) toevoeging. Het geeft een soort van vertrouwen dat het niet 'net nieuw' is ofzo. Misschien kunnen we dit toevoegen als: De community is ontstaan in 2017 toen bleek dat er veel overlap was in de vele online diensten die overheidsorganisaties aanbieden. Die overlap maakt samenwerken logisch en dat doen wij als community..... etc.

Als organisatie die meewerkt aan NL Design System beslis je dus mee over nieuwe ‘bouwblokken’, profiteer je van wat er al is en maak je gebruik van de expertise van het kernteam én de rest van de community.

## Welke NLDS-richtlijnen en componenten kan ik nu direct gebruiken om mijn website digitaal, gebruiksvriendelijk en toegankelijk te krijgen?

In de community zijn al veel verschillende [richtlijnen](/richtlijnen) en [componenten](/componenten) te vinden. De richtlijnen zijn direct op deze website te vinden, voor de componenten kun je terecht bij de organisaties waar ze gemaakt zijn. Op de pagina [componenten](/componenten) vind je links naar de relevante Storybooks en Figma-bestanden.

> Voor vragen, design of development: meld je gerust in de [Slack-community van Code for NL](https://praatmee.codefor.nl/) (kanaal `#nldesignsystem`).

Reviewer: Binnenkort kunnen we zoiets toevoegen:

Heb je een oplossing nodig voor een probleem van een specifiek WCAG succescriterium uit je toegankelijkheidsverklaring? Kijk dan op XXXX.

## Ik wil NL Design System gaan gebruiken voor mijn toegankelijkheidsopgave maar waar moet ik beginnen?

NL Design System helpt op dit moment op twee manieren bij de toegankelijkheidsopgave van organisaties:

> 1. Onze richtlijnen worden gebruikt als kennisbank. Ze bevatten concrete adviezen over wat er nodig is en waarom. Er zijn do's en don'ts, uitleg van de [WCAG-criteria](https://www.w3.org/Translations/WCAG21-nl/) en tips die verder dan gaan de WCAG.

Reviewer: Zou het niet tactisch zijn om het woord "kennisbank" te vermijden, ivm digitoegankelijk.nl?

Author: ```suggestion
1. Onze richtlijnen worden gebruikt als kennisbank. Ze bevatten concrete adviezen over wat er nodig is en waarom. Er zijn do's en don'ts, voorbeelden waarmee je aan[WCAG](https://www.w3.org/Translations/WCAG21-nl/) kunt voldoen en tips die verder dan gaan de WCAG.
```

> 1. Onze richtlijnen worden gebruikt als kennisbank. Ze bevatten concrete adviezen over wat er nodig is en waarom. Er zijn do's en don'ts, uitleg van de [WCAG-criteria](https://www.w3.org/Translations/WCAG21-nl/) en tips die verder dan gaan de WCAG.

Reviewer: In plaats van "uitleg van de WCAG-criteria" (dat is niet echt ons doel), denk ik dat we beter kunnen adverteren met "voorbeelden waarmee je kunt voldoen aan WCAG-criteria" of iets dergelijks.

> 2. Onze componenten kunnen worden gebruikt om toegankelijke producten te maken. Als componenten de laatste fase hebben bereikt (Hall of Fame), garanderen we zelfs dat ze voldoen aan toegankelijkheidseisen én zijn ze in productie getest.

Reviewer: Op de vraag "Waar moet ik beginnen" is het denk ik niet helpful om de niet bestaande hall of fame componenten te noemen, at this stage.

Author: Hm… leek me nuttig om de garantie waarover men misschien heeft gehoord te koppelen aan hall of fame…

## Mijn leverancier heeft vragen over NL Design System. Naar wie van jullie kan ik hem verwijzen?

Veel organisaties werken met NL Design System via hun leveranciers. Bijvoorbeeld doordat hun leveranciers onze componenten gebruiken, of meedoen in de community.

Ben je zelf leverancier en wil je voor klanten met NL Design System aan de slag? Neem dan contact op met het [kernteam](/project/kernteam), we helpen je graag op weg.

## Ik wil mijn manager voorstellen om mee te doen met NL Design System. Wat zijn de kosten en doorlooptijd? Waar moet ik zelf voor zorgen?

> Meedoen is gratis en kan stap voor stap. De kosten voor organisaties zitten in het beschikbaar stellen van resources en de tijd vrijmaken waarin ze samenwerken.

Reviewer: Beestje bij z'n naam noemen? "Tijd en geld"?

Author: zit in “kosten”? “tijd en geld vrijmaken waarin ze samenwerken” werkt niet?

Author: ```suggestion
Meedoen is gratis en kan stap voor stap. De kosten voor organisaties zitten in het beschikbaar stellen van mensen en de tijd vrijmaken waarin ze samenwerken.
```

Reviewer 2: Beter om resources te vervangen door mensen!

---
title: Veelgestelde vragen
hide_title: true
hide_table_of_contents: true
sidebar_label: Veelgestelde vragen
pagination_label: Veelgestelde vragen
description: Een overzicht van vragen die ons regelmatig gesteld worden.
keywords:
  - faq
  - nl design system
---

# Veelgestelde vragen

Op deze pagina geven we antwoord op vragen die regelmatig aan het kernteam van NL Design System gesteld worden.

- [Wat is het NL Design System?](#wat-is-het-nl-design-system)

> - [Hoe werkt het NL Design System en waarom?](#hoe-werkt-het-nl-design-system-en-waarom)

Author: ```suggestion
- [Hoe werkt het NL Design System](#hoe-werkt-het-nl-design-system)
```

---
title: Veelgestelde vragen
hide_title: true
hide_table_of_contents: false
sidebar_label: Veelgestelde vragen
pagination_label: Veelgestelde vragen
description: Een overzicht van vragen die ons regelmatig gesteld worden.
keywords:
  - faq
  - nl design system
---

# Veelgestelde vragen

> Het kernteam van NL Design System krijgt soms vragen die regelmatig terugkomen. Op deze pagina geven we op een aantal antwoord, zodat ze makkelijk terug te vinden zijn.

Reviewer 3: ```suggestion
Op deze pagina geven we antwoord op vragen die regelmatig aan het kernteam van NL Design System gesteld worden.
```

## Wat is het NL Design System?

NL Design System is een project dat specialisten van verschillende overheidsorganisaties laat samenwerken aan concrete oplossingen voor websites en applicaties. Denk bijvoorbeeld aan code, ontwerp en richtlijnen.

Het doel: organisaties hoeven minder vaak hetzelfde wiel uit te vinden. Zo worden online overheidsomgevingen consistenter, gebruiksvriendelijker en toegankelijker.

## Hoe werkt het NL Design System en waarom?

Het NL Design System bestaat uit verschillende soorten onderdelen:

> - [richtlijnen](/richtlijnen/introductie): adviezen over verschillende onderwerpen, zoals het [maken van formulieren](/richtlijnen/formulieren/).

Reviewer 3: ```suggestion
- [Richtlijnen](/richtlijnen/introductie): adviezen over verschillende onderwerpen, zoals het [maken van formulieren](/richtlijnen/formulieren/).
```

> - [componenten](/componenten): bouwblokken waarmee websites en applicaties kunnen worden opgebouwd, zoals knoppen, invulvelden en tabellen.

Reviewer 3: ```suggestion
- [Componenten](/componenten): bouwblokken waarmee websites en applicaties kunnen worden opgebouwd, zoals knoppen, invulvelden en tabellen.
```

> - [patronen](/voorbeelden/patronen): voorbeelden van hoe componenten samen kunnen worden gebruikt voor specifieke taken die gebruikers willen doen, op basis van inzichten uit [gebruikersonderzoek](/voorbeelden/onderzoek).

Reviewer 3: ```suggestion
- [Patronen](/voorbeelden/patronen): voorbeelden van hoe componenten samen kunnen worden gebruikt voor specifieke taken die gebruikers willen doen, op basis van inzichten uit [gebruikersonderzoek](/voorbeelden/onderzoek).
```

> - [handboek](/handboek/introductie): documentatie over het gebruik van NL Design System.

Reviewer 3: Deze de verwijzing naar Handboek zou ik, bovenaan zetten, weglaten of een andere plek geven. De reden hiervoor is de zin die hier direct achteraan komt. Het handboek ontstaat niet in de community.

```suggestion
- [Handboek](/handboek/introductie): documentatie over het gebruik van NL Design System.
```

Author: ```suggestion

Er is ook een [handboek](/handboek/introductie), documentatie over het gebruik van NL Design System.
```

> Deze onderdelen ontstaan niet bij het kernteam, maar in de community. Bijvoorbeeld omdat een gemeente of ministerie een specifiek component nodig heeft.

Reviewer 3: We spreken hier expliciet over 'component'. Maar hierboven benoemen we Richtlijnen, Componenten en Patronen. Zouden we dan het volgende moeten doen om het zo algemener te maken als enkel componenten?

```suggestion
Deze onderdelen ontstaan niet bij het kernteam, maar in de community. Bijvoorbeeld omdat een gemeente of ministerie een specifiek component of patroon nodig heeft.
```

> Als blijkt dat meer organisaties datzelfde component willen gebruiken, wordt er samengewerkt aan een ultieme versie, waarop het kernteam regie voert, kwaliteit bewaakt en garanties geeft. Dit doen we met het [estafettemodel](/handboek/estafettemodel)”. Hierin krijgen richtlijnen, componenten en patronen een status, die aangeeft hoe ver ze zijn gevorderd.

Reviewer 3: Zie comment hierboven. Ook hier spreken we enkel over 'component'.

Author: ```suggestion
Als blijkt dat meer organisaties datzelfde component of patroon willen gebruiken, wordt er samengewerkt aan een ultieme versie, waarop het kernteam regie voert, kwaliteit bewaakt en garanties geeft. Dit doen we met het [estafettemodel](/handboek/estafettemodel)”. Hierin krijgen richtlijnen, componenten en patronen een status, die aangeeft hoe ver ze zijn gevorderd.
```

> Als blijkt dat meer organisaties datzelfde component willen gebruiken, wordt er samengewerkt aan een ultieme versie, waarop het kernteam regie voert, kwaliteit bewaakt en garanties geeft. Dit doen we met het [estafettemodel](/handboek/estafettemodel)”. Hierin krijgen richtlijnen, componenten en patronen een status, die aangeeft hoe ver ze zijn gevorderd.

Author: ik maak er nog ff typographer's quotes van:

```suggestion
Als blijkt dat meer organisaties datzelfde component willen gebruiken, wordt er samengewerkt aan een ultieme versie, waarop het kernteam regie voert, kwaliteit bewaakt en garanties geeft. Dit doen we met het ‘[estafettemodel](/handboek/estafettemodel)’. Hierin krijgen richtlijnen, componenten en patronen een status, die aangeeft hoe ver ze zijn gevorderd.
```

## Wat is het estafettemodel? Hoe werkt dat model?

> De aanpak binnen NL Design System om samen te werken noemen we het “[estafettemodel](/handboek/estafettemodel)”. Hierin krijgen richtlijnen, componenten en patronen een status, die aangeeft hoe ver ze zijn gevorderd. Zo gaan onderdelen van [“Help Wanted”](/handboek/estafettemodel#help-wanted) naar [“Community”](/handboek/estafettemodel#community) naar [“Candidate”](/handboek/estafettemodel#candidate) naar [“Hall of Fame”](/handboek/estafettemodel#hall-of-fame). Elke stap heeft een eigen checklist, waarmee we kwaliteit waarborgen en bepalen wie wat doet.

Reviewer 3: ```suggestion
De aanpak binnen NL Design System om samen te werken noemen we het '[estafettemodel](/handboek/estafettemodel)'. Hierin krijgen richtlijnen, componenten en patronen een status, die aangeeft hoe ver ze zijn gevorderd. Zo gaan onderdelen van '[Help Wanted](/handboek/estafettemodel#help-wanted)' naar '[Community](/handboek/estafettemodel#community)' naar '[Candidate](/handboek/estafettemodel#candidate)' naar '[Hall of Fame](/handboek/estafettemodel#hall-of-fame)'. Elke stap heeft een eigen checklist, waarmee we kwaliteit waarborgen en bepalen wie wat doet.
```

Author: ```suggestion
De aanpak binnen NL Design System om samen te werken noemen we het ‘[estafettemodel](/handboek/estafettemodel)’. Hierin krijgen richtlijnen, componenten en patronen een status, die aangeeft hoe ver ze zijn gevorderd. Zo gaan onderdelen van ‘[Help Wanted](/handboek/estafettemodel#help-wanted)’ naar ‘[Community](/handboek/estafettemodel#community)’ naar ‘[Candidate](/handboek/estafettemodel#candidate)’ naar ‘[Hall of Fame](/handboek/estafettemodel#hall-of-fame)’. Elke stap heeft een eigen checklist, waarmee we kwaliteit waarborgen en bepalen wie wat doet.
```

> Elke stap heeft ook een eigen doel. De “Community” stap geeft organisaties alle vrijheid om te innoveren en van elkaar te gebruiken. In de “Candidate” stap is het onderdeel algemener gemaakt, breder ingezet en klaar voor de laatste feedback. Tot slot wordt het onderdeel definitief in de “Hall of Fame” stap, met garanties op toegankelijkheid en gebruiksvriendelijkheid.

Reviewer 3: ```suggestion
Elke stap heeft ook een eigen doel. De 'Community' stap geeft organisaties alle vrijheid om te innoveren en van elkaar te gebruiken. In de 'Candidate' stap is het onderdeel algemener gemaakt, breder ingezet en klaar voor de laatste feedback. Tot slot wordt het onderdeel definitief in de 'Hall of Fame' stap, met garanties op toegankelijkheid en gebruiksvriendelijkheid.
```

Author: ```suggestion
Elke stap heeft ook een eigen doel. De ‘Community’ stap geeft organisaties alle vrijheid om te innoveren en van elkaar te gebruiken. In de ‘Candidate’ stap is het onderdeel algemener gemaakt, breder ingezet en klaar voor de laatste feedback. Tot slot wordt het onderdeel definitief in de ‘Hall of Fame’ stap, met garanties op toegankelijkheid en gebruiksvriendelijkheid.
```

Door zo'n uitgebreid stappenplan te doorlopen, zorgen we voor een aantal zekerheden:

- **Goed onderzocht**: we nemen gebruikersonderzoek als basis voor bruikbare richtlijnen, componenten, patronen en betere toegankelijkheid.
- **Publiek beschikbaar**: we maken open source werken makkelijker, waardoor samenwerken makkelijk wordt.
- **Samen ontwikkeld**: oplossingen worden gebouwd en gebruikt door een actieve community.
- **Breed gedragen**: onderdelen zijn bij meerdere organisaties gebruikt en daardoor getest met zoveel mogelijk verschillende use cases.

> ## Waarom maken jullie niet gewoon zelf componenten, richtlijnen en ontwerpen en stellen die beschikbaar?

Reviewer 3: Eerder spreken we over richtlijnen, componenten en patronen. Dus dat ritme hier ook aanhouden? En dan minus 'Richtlijnen' omdat we die inmiddels toch echt wel zelf maken?

```suggestion
## Waarom maken jullie niet gewoon zelf componenten en patronen?
```

> We ontwerpen de onderdelen niet zelf, omdat we de doelen van deelnemende organisaties centraal willen stellen. En, nog belangrijker, de behoeften van hun gebruikers. Doordat specialisten in de community samenwerken stimuleren we vanaf het begin breed draagvlak. Zo worden oplossingen direct in de praktijk gebruikt en kunnen ze gelijk met de juiste doelgroep worden getest.

Reviewer 3: Met 'ontwikkelen' design en development te pakken?

```suggestion
We ontwikkelen de onderdelen niet zelf, omdat we de doelen van deelnemende organisaties centraal willen stellen. En, nog belangrijker, de behoeften van hun gebruikers. Doordat specialisten in de community samenwerken stimuleren we vanaf het begin breed draagvlak. Zo worden oplossingen direct in de praktijk gebruikt en kunnen ze gelijk met de juiste doelgroep worden getest.
```

> We ontwerpen de onderdelen niet zelf, omdat we de doelen van deelnemende organisaties centraal willen stellen. En, nog belangrijker, de behoeften van hun gebruikers. Doordat specialisten in de community samenwerken stimuleren we vanaf het begin breed draagvlak. Zo worden oplossingen direct in de praktijk gebruikt en kunnen ze gelijk met de juiste doelgroep worden getest.

Reviewer 3: We spreken hier trouwens over 'onderdelen'. Zou dat niet een mooi woord zijn om richtlijnen, componenten en patronen, continue mee samen te vatten in 1 woord?

Author: Ja, in veel gevallen wel! Zou het willen laten afhangen van of we ergens tactisch concreet willen zijn (en dus langer) of dat we liever kort zijn.

## Waarom hebben jullie een community? Wat doet die dan voor mij?

Overheidsorganisaties hebben veel overlap in de online diensten die ze aanbieden. Die overlap maakt samenwerken logisch en dat doen wij als community. Want als we dezelfde uitdagingen hebben, kunnen we vast ook van elkaar leren wat betreft oplossingen.

Als organisatie die meewerkt aan NL Design System beslis je dus mee over nieuwe ‘bouwblokken’, profiteer je van wat er al is en maak je gebruik van de expertise van het kernteam én de rest van de community.

## Welke NLDS-richtlijnen en componenten kan ik nu direct gebruiken om mijn website digitaal, gebruiksvriendelijk en toegankelijk te krijgen?

> In de community zijn al verschillende richtlijnen en componenten te vinden. Voor vragen, meld je gerust in de Slack-community van Code for NL.

Author: ```suggestion
In de community zijn al verschillende richtlijnen en componenten te vinden. Voor vragen, meld je gerust in de [Slack-community van Code for NL](https://praatmee.codefor.nl/) (kanaal `#nldesignsystem`).
```

> In de community zijn al verschillende richtlijnen en componenten te vinden. Voor vragen, meld je gerust in de Slack-community van Code for NL.

Author: @Robbert wat zou je aanraden als plek om naar te lnken als men 'verschillende componenten' wil vinden? is Utrecht storybook dan logisch of juist onze eigen componentenpagina waar links staan?

> ## Ik wil NL Design System gaan gebruiken voor mijn toegankelijkheidsopgave maar waar moet ik beginnen?

Author: Goede vraag, voor nu een shortcut voor “voldoen aan de wettelijke verplichtingen en klantvriendelijkheidsambitie en daar een proces voor hebben ingericht” maar da's te lang.

2. Onze componenten kunnen worden gebruikt om toegankelijke producten te maken. Als componenten de laatste fase hebben bereikt (Hall of Fame), garanderen we zelfs dat ze voldoen aan toegankelijkheidseisen én zijn ze in productie getest.

## Mijn leverancier heeft vragen over NL Design System. Naar wie van jullie kan ik hem verwijzen?

Veel organisaties werken met NL Design System via hun leveranciers. Bijvoorbeeld doordat hun leveranciers onze componenten gebruiken, of meedoen in de community.

Ben je zelf leverancier en wil je voor klanten met NL Design System aan de slag? Neem dan contact op met het [kernteam](/project/kernteam), we helpen je graag op weg.

## Ik wil mijn manager voorstellen om mee te doen met NL Design System. Wat zijn de kosten en doorlooptijd? Waar moet ik zelf voor zorgen?

Meedoen is gratis en kan stap voor stap. De kosten voor organisaties zitten in het beschikbaar stellen van resources en de tijd vrijmaken waarin ze samenwerken.

## Zijn er al andere organisaties die meedoen? Hebben jullie wat contactgegevens voor mij?

> NL Design System wordt gebruikt in verschillende projecten van overheidsorganisaties door heel Nederland. Veel van deze projecten zijn in ontwikkeling, sommige projecten staan live. [Op onze website](/community/wie-doet-mee/) vind je een overzicht van organisaties die zelf en/of via hun leveranciers meedoen met NL Design System.

Reviewer 4: Linktekst zetten op 'overzicht van organisaties' en niet op 'Op onze website'.

Author: ```suggestion
NL Design System wordt gebruikt in verschillende projecten van overheidsorganisaties door heel Nederland. Veel van deze projecten zijn in ontwikkeling, sommige projecten staan live. Op onze website vind je een [overzicht van organisaties](/community/wie-doet-mee/) die zelf en/of via hun leveranciers meedoen met NL Design System.
```

> NL Design System wordt gebruikt in verschillende projecten van overheidsorganisaties door heel Nederland. Veel van deze projecten zijn in ontwikkeling, sommige projecten staan live. Op onze website vind je een [overzicht van organisaties](/community/wie-doet-mee/) die zelf en/of via hun leveranciers meedoen met NL Design System.

Reviewer 3: Als je dit leest ben je al op 'onze website'

Author: ```suggestion
NL Design System wordt gebruikt in verschillende projecten van overheidsorganisaties door heel Nederland. Veel van deze projecten zijn in ontwikkeling, sommige projecten staan live. Op de pagina [‘Wie doet mee?’](/community/wie-doet-mee/) vind je een overzicht van organisaties die zelf en/of via hun leveranciers meedoen met NL Design System.
```

Meedoen kan op allerlei manieren, van alleen gebruiken tot zelf meebouwen:

> - **Gebruik de bouwstenen**: gebruik [richtlijnen](/richtlijnen), [componenten](/componenten) of [onderzoek](/onderzoek) voor je websites en apps.

Reviewer 4: linken naar /richtlijnen/introductie

Author: ```suggestion
- **Gebruik de bouwstenen**: gebruik [richtlijnen](/richtlijnen/introductie), [componenten](/componenten) of [onderzoek](/onderzoek) voor je websites en apps.
```

> - **Gebruik de bouwstenen**: gebruik [richtlijnen](/richtlijnen/introductie), [componenten](/componenten) of [onderzoek](/onderzoek) voor je websites en apps.

Reviewer 3: Eerder schrijven we 'websites en applicaties' hier dan ook?

```suggestion
- **Gebruik de bouwstenen**: gebruik [richtlijnen](/richtlijnen/introductie), [componenten](/componenten) of [onderzoek](/onderzoek) voor je websites en applicaties.
```

- **Doe kennis op**: kom naar bijeenkomsten als de tweewekelijkse Heartbeat, Design Systems Week of hackatons. Of word lid van onze [Slack community bij CodeForNL](https://praatmee.codefor.nl/) (kanaal: #nl-design-system).
- **Werk mee**: deel je eigen richtlijnen, componenten of onderzoek, of geef feedback op de bestaande bouwstenen.

## Hoe is security geregeld binnen NLDS?

> Het kernteam beheert de _repository's_ waarin de diverse NL Design System projecten publiek worden ontsloten, zoals thema's en organisatie-specifieke design systems.

Reviewer 4: _GitHub repositories_
link naar https://github.com/orgs/nl-design-system/repositories

Author: ```suggestion
Het kernteam beheert de [_repository's_](https://github.com/orgs/nl-design-system/repositories) waarin de diverse NL Design System projecten publiek worden ontsloten, zoals thema's en organisatie-specifieke design systems.
```

Reviewer 4: repository's is fout Engels :-) 
it's repositories

Author: het is correct Nederlands blijkbaar, zoals [community's](https://www.digitaleoverheid.nl/communitys/). vind het zelf ook niet echt een gezicht moet ik zeggen.

Author: Ja je hebt helemaal gelijk, dat suggereert dat het Engels is… nu niet meer schuin!

> Het kernteam beheert de [_repository's_](https://github.com/orgs/nl-design-system/repositories) waarin de diverse NL Design System projecten publiek worden ontsloten, zoals thema's en organisatie-specifieke design systems.

Reviewer 3: Dit stond italic?

```suggestion
Het kernteam beheert de [repository's](https://github.com/orgs/nl-design-system/repositories) waarin de diverse NL Design System projecten publiek worden ontsloten, zoals thema's en organisatie-specifieke design systems.
```

> We passen best practices toe om te zorgen dat deze repository's up to date en veilig blijven, zoals automatisch instellen van rechten, _branch protection_ en procedures voor het regelmatig updaten van dependency's.

Reviewer 3: Dit stond italic?

```suggestion
We passen best practices toe om te zorgen dat deze repository's up to date en veilig blijven, zoals automatisch instellen van rechten, branch protection en procedures voor het regelmatig updaten van dependency's.
```
